tests/kernel/sched/preempt/testcase.yaml

```diff
@@ -1,8 +1,9 @@
 tests:
   kernel.scheduler.preempt:
     tags: kernel
-    filter: not CONFIG_SMP
     platform_exclude: nrf52dk_nrf52810
+    extra_configs:
+      - CONFIG_SMP=n
```

Comment:
Hmm, as I remember we do not disable SMP from the test suit. If we need execution only on one core we set `CONFIG_MP_NUM_CPUS` to 1.

Reply:
> Hmm, as I remember we do not disable SMP from the test suit. If we need execution only on one core we set `CONFIG_MP_NUM_CPUS` to 1.

This original test suite works when CONFIG_SMP is not enabled. Currently when the board configure as CONFIG_SMP by default for all other testing, this testing case will be skipped. After I do these changes so that the board by default with CONFIG_SMP can run this case.

Reply:
> Hmm, as I remember we do not disable SMP from the test suit. If we need execution only on one core we set `CONFIG_MP_NUM_CPUS` to 1.

This original test suite works when CONFIG_SMP is not enabled. Currently when the board configure as CONFIG_SMP by default for all other testing, this testing case will be skipped. After I do these changes so that the board by default with CONFIG_SMP can run this case.

Reply:
Yeah, this filter is ancient and basically wrong in today's environment where we have lots of SMP platforms (which weren't getting coverage in this test!).  Agree we should remove it.

But also agreed: the "extra_configs" here isn't doing anything useful.  There's no need to set SMP=n, we just need to know there's only one cpu (which is done in prj.conf already).

Reply:
> Yeah, this filter is ancient and basically wrong in today's environment where we have lots of SMP platforms (which weren't getting coverage in this test!). Agree we should remove it.
> 
> But also agreed: the "extra_configs" here isn't doing anything useful. There's no need to set SMP=n, we just need to know there's only one cpu (which is done in prj.conf already).

Thank you Andy. I will fix as you said.